conftest.py

```diff
@@ -84,7 +84,7 @@ def pytest_collection_modifyitems(config, items):
         ]
 
         if os.name == 'nt':
-            percentage = 0.5
+            percentage = 0.4
```

Comment:
Times are still high for the slow slots. Perhaps you should reduce this to `0.2` or `0.1`.